azure-fluid-relay-preview-pr/articles/howtos/hosting.md

```diff
@@ -0,0 +1,258 @@
+    storage: "https://myStorageUrl",
+};
+const client = new AzureClient(config);
+```
+
+<br />
+
+> [!NOTE]
+> The below implementation can be used for `Static Web Apps` only.
+
+3. `Static Web Apps` allow you to develop a full-stack web site without needing to deal with the server-side configuration of an entire web hosting environment and not exposing the token on the client-side bundle code. You can fetch the token by setting up your own backend `api`. Developers can create their own `HTTPTrigger Azure Function` under an `api` folder in the `FluidHelloWorld` repository itself. Below, `GetFrsToken` is an `HTTPTrigger Azure Function` which is responsible for fetching token from your very own backend. 
+
+<img src='./resources/images/apiFileExplorer.png' style="height: 300px"> 
+
+<br />
+<br />
+
+The below code snippet can help you with creating `HTTPTrigger` function for fetching the token by passing in your tenant key.
+
+```typescript
+import { AzureFunction, Context, HttpRequest } from "@azure/functions";
+import { ScopeType } from "@fluidframework/protocol-definitions";
+import { generateToken, generateUser } from "@fluidframework/server-services-utils";
+
+const key = "myTenantKey";
+
+const httpTrigger: AzureFunction = async function (context: Context, req: HttpRequest): Promise<void> {
+    const tenantId = (req.query.tenantId || (req.body && req.body.tenantId)) as string;
+    const documentId = (req.query.documentId || (req.body && req.body.documentId)) as string;
+    const userId = (req.query.userId || (req.body && req.body.userId)) as string;
+    const userName = (req.query.userName || (req.body && req.body.userName)) as string;
+    const scopes = (req.query.scopes || (req.body && req.body.scopes)) as ScopeType[];
+
+    if (!tenantId) {
+        context.res = {
+            status: 400,
+            body: "No tenantId provided in query params",
+        };
+    }
+
+    if (!key) {
+        context.res = {
+            status: 404,
+            body: `No key found for the provided tenantId: ${tenantId}`,
+        };
+    }
+
+    if (!documentId) {
+        context.res = {
+            status: 400,
+            body: "No documentId provided in query params"
+        };
+    }
+
+    let user = { name: userName, id: userId };
+    if (!userId || !userName) {
+        const generatedUser = generateUser() as any;
+        user = { name: userName ?? generatedUser.name, id: userId ?? generatedUser.id };
+    }
+
+    const token = generateToken(
+        tenantId,
+        documentId,
+        key,
+        scopes ?? [ScopeType.DocRead, ScopeType.DocWrite, ScopeType.SummaryWrite],
+        user
+    );
+
+    context.res = {
+        status: 200,
+        body: token
+    };
+};
+
+export default httpTrigger; 
+```
+
+<br />
+
+In this case, you would also have to add `key` as a parameter in your App Settings. You can do so by navigating to `Configuration` in `Settings` tab and selecting `Add` application setting. 
```

Comment:
Is this still necessary since it is a constant in the code itself? Perhaps the constant value could be fetched from the app settings. That way it wouldn't need to get checked in with the code and this line would also be valid